modules/ROOT/pages/scatter-gather-concept.adoc

```diff
@@ -3,7 +3,11 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-The Scatter-Gather component is a routing event processor that processes a Mule event through different parallel processing routes that contain one or more event processors. Each route receives a reference to the Mule event and executes a sequence of one or more event processors. Each of these routes uses a separate thread and creates a new Mule event with its own payload, attributes, and variables. The Scatter-Gather component then combines the Mule events returned by each processing route into a new Mule event that is passed to the next event processor only after every route completes successfully.
+The Scatter-Gather component is a routing event processor that processes a Mule event through different parallel processing routes that contain one or more event processors. Each route receives a reference to the Mule event and executes a sequence of one or more event processors. Each of these routes uses a separate thread to execute the event processors, and the resulting Mule event varies according to the following:
+* If the processors in the route apply any modification to the Mule event received, the processors create a new Mule event with its own payload, attributes, and variables. When the route completes, it returns the new Mule event.
```

Comment:
this is general to all mule chains, i don't think it is necessary to make it explicit here for scatter gather (unless we are clarifying this for all routes, which i don't think is the case)